README.md

```diff
@@ -8,99 +8,166 @@ _user_ not a _developer_.
 
 What's the difference? Well consider a mechanic driving to work. In that
 moment, waiting in traffic, they are a _user_ of the car. But then they take a
-right turn and *cough*, *splutter* their car chokes to a stop. At once their
-mind changes to being a _mechanic_ or a _developer_. They think about
+right turn and *cough*, *splutter* &mdash; their car chokes to a stop. At once
+their mind changes to being a _mechanic_ or a _developer_. They think about
 petroleum, spark-plugs and ignitions. They can _debug_ their car using
 techniques they've learned. They can imagine what might be wrong. To get more
 data or to verify their guess, they _open up the hood_. Learning to be a
 developer (_computer mechanic_?) is similar. You must start "looking under the
 hood" of your computer. To access the "hood's" contents, you must use the
-shell.
+terminal.
```

Comment:
I like the car metaphor, and I think it's universal enough to be relevant to our student body, although we definitely have a more urban population.  It works nicely to illustrate the transition between turning from a user into a debugger.